ui/components/DashboardComponent.js

```diff
@@ -1229,6 +1229,7 @@ class DashboardComponent extends React.Component {
   };
 
   render() {
+    console.info("Rerendering Dashboard")
```

Comment:
@aabidsofi19 please remove this console if not needed.

Reply:
I thought to keep it for an easy check to see if the dashboard rerenders are really increasing in production  .